Azure/azure-cli
# [RDBMS] `az postgres flexible-server replica create`: Allow read replica create for flexible server with storage auto-grow as Enabled

**Related command**
`az postgres flexible-server replica create`

**Description**
Adding support to have storage auto-grow enabled servers to be able to create read replica.

**Testing Guide**
Manual
SAG enabled source server should be able to create replica with SAG enabled:
(env) PS C:\Users\nasc\azureCLI\azure-cli> az postgres flexible-server replica create --replica-name nasc-rep-sag-cli-1 -g rg --source-server nasc-sag1018
{
  ...
  "fullyQualifiedDomainName": "nasc-rep-sag-cli-1.postgres.database.azure.com",
  ...
  "replica": {
    "capacity": 0,
    "promoteMode": null,
    "promoteOption": null,
    "replicationState": null,
    **"role": "AsyncReplica"**
  },
...
  "state": "Ready",
  "storage": {
    **"autoGrow": "Enabled",**
    "iops": 500,
    "storageSizeGb": 128,
    "throughput": null,
    "tier": "P10",
    "type": ""
  },

**History Notes**
[RDBMS] `az postgres flexible-server replica create`: Allow read replica create from storage auto-grow enabled primary server flexible server
